client/context.go

```diff
@@ -71,7 +71,7 @@ func (ctx Context) WithKeyringOptions(opts ...keyring.Option) Context {
 
 // WithInput returns a copy of the context with an updated input.
 func (ctx Context) WithInput(r io.Reader) Context {
-	// convert to a bufio.Reader to have a shared buffer between the keyring and the
+	// convert to a bufio.Reader to have a shared buffer between the keyring and 
```

Comment:
@omahs , please check lint error.

Reply:
@zemyblue Hi, I'm not a dev I don't know how to fix such error, I can remove the problematic commit if helpful